src/patternfly/components/Masthead/masthead.scss

```diff
@@ -0,0 +1,220 @@
+  --pf-c-masthead__main--MarginRight: var(--pf-c-masthead--m-display-inline__main--MarginRight);
+  --pf-c-masthead__main--before--BorderBottom: var(--pf-c-masthead--m-display-inline__main--BorderBottom);
+  --pf-c-masthead__content--Order: var(--pf-c-masthead--m-display-inline__content--Order);
+  --pf-c-masthead__content--PaddingTop: var(--pf-c-masthead--m-display-inline__content--PaddingTop);
+  --pf-c-masthead__content--PaddingBottom: var(--pf-c-masthead--m-display-inline__content--PaddingBottom);
+  --pf-c-masthead__content--MarginLeft: var(--pf-c-masthead--m-display-inline__content--MarginLeft);
+  --pf-c-masthead__content--c-nav--m-horizontal--MarginRight: var(--pf-c-masthead--m-display-inline__content--c-nav--m-horizontal--MarginRight);
+  --pf-c-masthead__content--c-nav--m-horizontal--MarginLeft: var(--pf-c-masthead--m-display-inline__content--c-nav--m-horizontal--MarginLeft);
+}
+
+.pf-c-masthead {
+  --pf-c-masthead--PaddingLeft: var(--pf-c-masthead--inset);
+  --pf-c-masthead--PaddingRight: var(--pf-c-masthead--inset);
+  --pf-c-masthead--BackgroundColor: var(--pf-global--BackgroundColor--dark-100);
+  --pf-c-masthead--inset: var(--pf-global--spacer--lg);
+
+  // insets
+  --pf-c-masthead--inset: var(--pf-global--spacer--md); // match page insets
+  --pf-c-masthead--xl--inset: var(--pf-global--spacer--lg); // match page insets
+
+  // stack
+  --pf-c-masthead--m-display-stack--FlexWrap: wrap;
+  --pf-c-masthead--m-display-stack__main--Order: -1;
+  --pf-c-masthead--m-display-stack__main--FlexBasis: 100%;
+  --pf-c-masthead--m-display-stack__main--PaddingTop: var(--pf-global--spacer--sm);
+  --pf-c-masthead--m-display-stack__main--PaddingBottom: var(--pf-global--spacer--sm);
+  --pf-c-masthead--m-display-stack__main--MarginRight: 0;
+  --pf-c-masthead--m-display-stack__main--before--BorderBottom: var(--pf-c-masthead__main--before--BorderBottomWidth) solid var(--pf-c-masthead__main--before--BorderBottomColor);
+  --pf-c-masthead--m-display-stack__content--Order: 1;
+  --pf-c-masthead--m-display-stack__content--PaddingTop: var(--pf-global--spacer--sm);
+  --pf-c-masthead--m-display-stack__content--PaddingBottom: var(--pf-global--spacer--sm);
+  --pf-c-masthead--m-display-stack__content--MarginLeft: 0;
+  --pf-c-masthead--m-display-stack__content--c-nav--m-horizontal--MarginRight: calc(var(--pf-c-masthead--inset) * -1);
+  --pf-c-masthead--m-display-stack__content--c-nav--m-horizontal--MarginLeft: calc(var(--pf-c-masthead--inset) * -1);
+
+  // inline
+  --pf-c-masthead--m-display-inline--FlexWrap: nowrap;
+  --pf-c-masthead--m-display-inline__main--Order: 0;
+  --pf-c-masthead--m-display-inline__main--FlexBasis: auto;
+  --pf-c-masthead--m-display-inline__main--PaddingTop: 0;
+  --pf-c-masthead--m-display-inline__main--PaddingBottom: 0;
+  --pf-c-masthead--m-display-inline__main--MarginRight: calc(var(--pf-global--gutter--md) / 2); // split gutter
```

Comment:
Why does this use the gutter global var over the large spacer? Or is it meant to be half of whatever the current inset is? (24px > 1200px viewport, 16px < 1200px)?